### ISSUE TITLE
whatapp block me after 10 message !

### ISSUE THREAD

**Issue by amregbaly** [venomous0x/WhatsAPI#757]
Hello !
i developing script for Institute i send a message every 5 mins .. after the 10 message the number get block !!!
just 10 messages .. iam not a spammer .. no one block my number i guess .. 
any solutions !?

**Comment by momenimohammad23** [comment]
@mgp25  hi
I need to a document to use this API :(
may you help me plz

**Comment by creolyte** [comment]
P/S: I do not know that when using WhatsAPI you could not use in your phone.

Hence, when I was testing the WhatsAPI , then I went to open my mobile Whasapp and it prompt me to re-verify my existence and I did, to check my messages. After that I went to the API to try send 1 message again but my connection appeared to be blocked!!!
Is this a permanent block of using WhatsAPI? I need to use other number to try now?? zzz
Please help...

**Comment by hpatel4u** [comment]
You can not use the same number in mobile and in API at the same time, your number may be blocked by whatsapp. share your API log which helps to know exact status.

**Comment by creolyte** [comment]
Thank you for your reply hpatel4u. Where can I find my logs? Do you mean the error returned by the codes? By the way, I got this error below earlier:
</>failure</>
</>not-authorized>
</>failure</>

and I read the FAQ section that you are permanently blocked if I get this. By the way, how to switch back to mobile Whatsapp (without getting blocked again) because I need to check my messages. Or the best option is get a new number for development purposes? Thanks.

**Comment by hpatel4u** [comment]
Yes your number blocked by Whatsapp.

**Comment by creolyte** [comment]
Means game over? I need get a new sim card with a new number for development again everytime got blocked?

**Comment by mgp25** [comment]
@creolyte @hpatel4u or maybe using an outdated API because this one is outdated, use Chat-API instead

**Comment by creolyte** [comment]
Hi guys, thanks for the replies. Yup I'm using Chat-API. All I did was:

1) Request code and successfully got the password
2) Then I register ($w->codeRegister( $code );)
3) I went back to my mobile to check my whatsapp chat and it prompt me to re-verify and I did.
4) After checking my personal message, I went back to development trying to do anything but was blocked.
5) Tried re-verify by getting new code and password from API still can't work (still blocked).

Hope this help future people after reading this. 
1) Is there any way to unblock?
2) Once connected to API means I can't check my personal message on mobile phone? (Because once I did that and I instantly got blocked lol?)

**Comment by hpatel4u** [comment]
@creolyte you can not logged in to two different place at the same time and do not try to request code more then two or three time in a day. Whatsapp tracing this also very well.

@mgp25 I am using the API which was working fine before 10days and now the blockage ration is very high. if you want i can share you the code you can review on your time and suggest at your time.

FYI, i just changed the token, version etc from Chat-API but there is no success.

**Comment by creolyte** [comment]
@hpatel4u There is no logout function from the mobile whatsapp :(

But thanks I'll keep in mind and try update any new findings here for future developers benefits.

**Comment by mgp25** [comment]
Working every nice on my side.

By the way, im not answering anymore in the repository as its outdated an inactive

Regards

**Comment by AndantioKorah** [comment]
so if i use phone number A (example) for whatsapi, then, i must dont use phone number A in my phone?. So how about when i got my CodeRequest and Password from WART and then whatsapp request for verify my account in phone. I should verify it or not?
please help me, i'm beginner at this thing.

**Comment by creolyte** [comment]
Yes you can and you will able to access to whatsapp from your mobile again. But what happened to me is that after that i went back to connect to whatsapi again, it blocked me. Maybe is because i request too many times in a day? All the best.

-------- Original message --------

**Comment by AndantioKorah** [comment]
done, i made it. I try send to my phone and work and again send to my friend's whatsapp and it work again. But when i try again send message to my number, it doesn't work anymore . Anybody know why? How about if i delete my whatsapp app in phone, it will work?

**Comment by foratech** [comment]
I found a solution for one whatsapp number to be able to send out to multiple recipient to avoid being blocked!! email me to know the answer.. azizi@foratechsolution.com, I wont post the answer here..

**Comment by mgp25** [comment]
Do **NOT** trust @foratech is a scammer. FAKE

Check only trusted sources:https://github.com/WHAnonymous/Chat-API

**Comment by foratech** [comment]
lol.. I'm using the same API, I'm not scamming.. I'm using the same API but applying different algo to avoid the number being block that all... depends if you believe me or not.. anyone can email me to see the demo website I made.. using that API. one number will be able to send out to 100 different recipient without being blocked..

**Comment by AndantioKorah** [comment]
how about to send file attachment? can i send it or not? please answer ASAP. please

**Comment by Muruganpari** [comment]
Hi.. peps, i am using whatsapp api script, one month before working condition is correctly but still some problems are there, few days before my number is blocked, i don't know what is the reasson, after i found some results for stright flow running api then sendMessageComposing(), sendMessagePaused() these two events are added then still number is blocked, i think some security hole is there so please give me the solution anybody.....

@mgp25 plese give me the solution..

**Comment by Harshaddarji** [comment]
jkhkj

**Comment by Harshaddarji** [comment]
From Last four days my bumber getting blocked over Whatsapp.

Is Any one facing sam issue recently  ? 

any update?

**Comment by janemarisca** [comment]
i want to try it, where can i find the code? thank you

**Comment by assegaf** [comment]
@foratech  is fake scammer, I have implemented WA20 protocol, which is based on Android Whatsapp 2.16.xxx, and 2.12.5xx version, everything is fine, if you are using user agent that version and still using WA16 protocol, not sure but random block even not doing abusing system even a little,
